src/main/java/ChatReques6.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatReques6 {
```

Comment:
Using wildcard types like List<?> can lead to type safety issues as it allows any type of object to be added to the list. Consider specifying the type explicitly to ensure type safety.